cmd/wallet-web/src/components/Modal/Modal.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div
+    v-show="isOpen"
+    class="flex overflow-y-auto fixed inset-0 z-50 justify-center items-center bg-gradient-harold"
+    :class="{ open: isOpen }"
+    @click="$emit('onClose')"
+  >
+    <div
+      class="relative mx-6 lg:mx-auto max-w-6xl bg-neutrals-white rounded-2xl modal-width"
+      :class="{ open: isOpen }"
+      @click.stop
+    >
+      <div v-if="title" class="pt-5 pb-3 text-lg font-bold text-neutrals-dark">{{ title }}</div>
+      <div>
+        <slot />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Modal',
+  props: {
+    isOpen: Boolean,
+    title: String,
```

Comment:
Could you add default values for optional props or declare them as required instead, please?